src/style.css

```diff
@@ -0,0 +1,80 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-left: 4px;
+
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:first-child {
+  margin: 0;
+}
+
+.star--0 :nth-child(1),
+.star--0 :nth-child(2),
+.star--0 :nth-child(3),
+.star--0 :nth-child(4),
+.star--0 :nth-child(5) {
+  background-image: url(images/star.svg);
+}
+
+.star--1 :nth-child(1) {
+  background-image: url(images/star-active.svg);
+}
+
+.star--1 :nth-child(2),
+.star--1 :nth-child(3),
+.star--1 :nth-child(4),
+.star--1 :nth-child(5) {
+  background-image: url(images/star.svg);
+}
+
+.star--2 :nth-child(1),
+.star--2 :nth-child(2) {
+  background-image: url(images/star-active.svg);
+}
+
+.star--2 :nth-child(3),
+.star--2 :nth-child(4),
+.star--2 :nth-child(5) {
+  background-image: url(images/star.svg);
+}
+
+.star--3 :nth-child(1),
+.star--3 :nth-child(2),
+.star--3 :nth-child(3) {
+  background-image: url(images/star-active.svg);
+}
+
+.star--3 :nth-child(4),
+.star--3 :nth-child(5) {
+  background-image: url(images/star.svg);
+}
+
+.star--4 :nth-child(1),
+.star--4 :nth-child(2),
+.star--4 :nth-child(3),
+.star--4 :nth-child(4) {
+  background-image: url(images/star-active.svg);
+}
+
+.star--4 :nth-child(5) {
+  background-image: url(images/star.svg);
+}
+
+.star--5 :nth-child(1),
+.star--5 :nth-child(2),
+.star--5 :nth-child(3),
+.star--5 :nth-child(4),
+.star--5 :nth-child(5) {
+  background-image: url(images/star-active.svg);
```

Comment:
come on, these are the same too

Reply:
Shortened css even more with nth-child(n)

Reply:
Tests were failing because my class name was star--# instead of stars--# 
Now solved.